charts/prometheus-elasticsearch-exporter/templates/podsecuritypolicies.yaml

```diff
@@ -1,9 +1,6 @@
 {{- if .Values.podSecurityPolicies.enabled -}}
-{{- if .Capabilities.APIVersions.Has "policy/v1/PodSecurityPolicy" }}
-apiVersion: policy/v1
-{{- else }}
+{{- if .Capabilities.APIVersions.Has "policy/v1beta1/PodSecurityPolicy" }}
```

Comment:
This could be a single line:
```
{{- if and .Values.podSecurityPolicies.enabled .Capabilities.APIVersions.Has "policy/v1beta1/PodSecurityPolicy" }}
```